dbx/jvm/v5.2-wn-items.rst

```diff
@@ -18,6 +18,21 @@
   the same ``mongos`` server for operation retry attempts if other ``mongos``
   servers are available.
 
+- Enables exact vector search by replacing the ``VectorSearchOptions``
+  type with the following types:
```

Comment:
The API for VectorSearchOptions has been changed by introducing more specific option subtypes for use
ApproximateVectorSearchOptions and ExactVectorSearchOptions.

Reply:
Will update the wording!